samples/js/node/notebooks/004-hello-detection-with-output.nnb

```diff
@@ -0,0 +1,161 @@
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Do Inference"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "const tensorData = new Float32Array(inputImage);\nconst tensor = new ov.Tensor(ov.element.f32, Int32Array.from(shape), tensorData);\n\nconst inferRequest = compiledModel.create_infer_request();\ninferRequest.set_input_tensor(tensor);\ninferRequest.infer();\n\nconst output = inferRequest.getTensor(outputLayer);\n\nconsole.log(output.data);"
```

Comment:
`shape` is not defined. You probably meant `inputLayer.shape`
`const tensor = new ov.Tensor(ov.element.f32, Int32Array.from(inputLayer.shape), tensorData);`